terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
+  storage_type = "io1"
+  identifier = "fec-govcloud-prod"
```

Comment:
AWS won't let you change a database identifier without destroying and recreating, which terraform captures using the `ForceNew` flag: https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_db_instance.go#L103-L109. We're setting the `prevent_destroy` lifecycle flag in this file to prevent potentially unintended destroys and recreates, so AFAIK this change will cause terraform to crash--it wants to destroy the database because the identifier changed, but it can't because we told it never to destroy these resources. If changing the identifier is important (I'm guessing it isn't, because we're already setting the database names), then you can also temporarily drop the `prevent_destroy` flags, then restore them after applying these changes.